### ISSUE TITLE
Current install fail ? sha256sum is incorrect

### ISSUE THREAD

**Issue by alexAubin** [YunoHost-Apps/lionwiki-t2t_ynh#13]
Passing by on this repo because of the issue reported here https://forum.yunohost.org/t/lionwiki-fails-install/17930

Not sure what version was being exactly, but the checksum seems indeed incorrect, I'm getting `5b7397da673c63f0207a5a832c33f3181e715948ffcf47d95d85bd7231215eda` from https://sourceforge.net/projects/lionwiki-t2t/files/lionwiki-t2t.zip compared to `3b893ed663da7fb4db381e88d5044281452c68fe3ad920b5a23b326ce5a2e4f6` which is the expected value according to app.src ...

Moreover, the URL https://sourceforge.net/projects/lionwiki-t2t/files/lionwiki-t2t.zip seems to point to the latest version available which isnt super great considering the install is going to spontaneously break again in the future when a new version is released and therefore the checksum will need to be upgraded again ... I'm not sure to understand the comment here https://github.com/YunoHost-Apps/lionwiki-t2t_ynh/pull/11/files/4043d029b6e148821f30c63881ee63e2761f3317#diff-3a108ffdade1415d57c1c60850a80f5452359bb8e1bf6c57f99211d45de7f6eb regarding Eric G's proposal x_X

**Comment by farvardin** [comment]
@alexAubin I've commited it correctly there: https://github.com/YunoHost-Apps/lionwiki-t2t_ynh/commit/87673e01a8dfd5cfc51f92da0e3d74370d79ca84

but I made a mistake while integrating a PR and I've put back a wrong sha256sum.

There is no tagged release on the sourceforge version, but every time I put a new version on SF, I correct the yunohost package (I do both packages). 

Mon commentaire dit exactement ce qui est expliqué : lionwiki sur lionwiki.0o.cz et lionwiki-t2t sur sourceforge.net/projects/lionwiki-t2t sont 2 projets différents (le second est un fork du premier). La version packagée pour yunohost prend en compte uniquement lionwiki-t2t. Si on met à la place la version sur lionwiki.0o.cz, les scripts d'installation ne sont pas prévus pour et ça ne fonctionnera probablement pas.
